### ISSUE TITLE
git clone pulls weight files as well by default

### ISSUE THREAD

**Issue by beasteers** [marl/openl3#70]
I wonder if it would make sense to move the model files to their own repo or if there's a way to not have them pull by default.

Basically, this is a tad annoying haha.
```
Cloning into 'openl3'...
remote: Enumerating objects: 1460, done.
remote: Counting objects: 100% (44/44), done.
remote: Compressing objects: 100% (36/36), done.
Receiving objects:  24% (357/1460), 444.15 MiB | 2.55 MiB/s  
```

**Comment by auroracramer** [comment]
Yeah, this is pretty annoying. You can use `--branch <branch-name> --single-branch` when cloning to only clone master. We can add it to the README for now.

Longer term, I'm not sure where the best place to move it is. Hosting model files on GitHub is already kind of messy and frowned upon (and is limited to 100MB), though since the weight files aren't too big, maybe it's fine moving it to another repo? As far as other platforms, hosting on Google Drive seems risky as far as access and ownership. IIRC I've seen Zenodo used for model releases, maybe that could work?

Either way, moving the models would break all of the previous versions, so maybe we could do another symlink-like hack to redirect to the new location. 

idk what do you think?
